src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingService.java

```diff
@@ -187,26 +187,69 @@ private String initialRespondentDetailsMarkUp(CaseData caseData) {
 
     private void populateRespondentAcasDetailsMarkUp(CaseData caseData) {
         List<RespondentSumTypeItem> respondentList = caseData.getRespondentCollection();
```

Comment:
I agree but check style and sonar doesn't, sonar has refactored the code like this. If you have an idea to remove the duplicated code without failing switch, it would be great

Reply:
switch case is there wit falling back warning suppressed as discussed